extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -319,6 +349,61 @@ export class AmpSelector extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Handles toggle action.
+   * @param {number} index
+   * @param {boolean} value
```

Comment:
Typically for toggle functions the `value` parameter is optional. This will help eliminate the branches in the action callback.

Reply:
You can base your toggle implementation on the `toggleExperiments` function in [`src/experiments.js`
](https://github.com/ampproject/amphtml/blob/master/src/experiments.js#L185)

Reply:
Changed the code.

Reply:
Hmm, I was thinking that this method would have `@param {boolean=} opt_value`, then in the `registerAction` callback just call `this.toggle_(args['index'], args['value'])` since you're checking `hasAttribute` and if `value` is undefined twice.